### PR TITLE
change api-journey to journeys

### PR DESCRIPTION
**Pull request by tataihono**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ We recommend using [Visual Studio Code](https://code.visualstudio.com/) with the
 As an example we are going to run through the steps to get the Next Steps Journeys project running in your web browser. This example assumes you are running it from the development container.
 
 1. Make a copy of `apps/api-gateway/.env` with the file name `apps/api-gateway/.env.local`. Open the file and set up your own environment variables.
-1. Make a copy of `apps/api-journey/.env` with the file name `apps/api-journey/.env.local`. Open the file and set up your own environment variables.
+1. Make a copy of `apps/journeys/.env` with the file name `apps/journeys/.env.local`. Open the file and set up your own environment variables.
 1. run `nx run api-journeys:migrations`
 1. run `nx run api-journeys:seed`
 1. run `nx run api-gateway:serve-all`
```
Comment (csiyang) [inline CONTRIBUTING.md:91]: Can we update this while we're here?

```suggestion
 run `nx run journeysy:serve-all`
```

**Review by csiyang — CHANGES_REQUESTED** [review]
Can we update this to have `nx run journeys:serve-all` while we're here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tataihono) [reply CONTRIBUTING.md:91]: err.... this is correct. You need to start the gateway.

**Review by csiyang — APPROVED** [review]
looks good!

**Pull request merged**